controllers/controller_volume_statefulset.go

```diff
@@ -16,7 +16,7 @@ func buildVolumeServerStartupScript(m *seaweedv1.Seaweed, dirs []string) string
 	commands := []string{"weed", "volume"}
 	commands = append(commands, fmt.Sprintf("-port=%d", seaweedv1.VolumeHTTPPort))
 	commands = append(commands, "-max=0")
-	commands = append(commands, fmt.Sprintf("-ip=$(POD_NAME).%s-volume-peer", m.Name))
+	commands = append(commands, fmt.Sprintf("-ip=$(POD_NAME).%s-volume-peer,%s", m.Name, m.Namespace))
```

Comment:
there is a bug here https://github.com/seaweedfs/seaweedfs-operator/issues/37

Reply:
Oh, sorry... I will fix and recheck all.

Reply:
Next time will be testing more changes... We decided to move all our data to seaweedfs and will be starting to contribute.